extra/restart-datadog-agent.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+if [[ $DISABLE_DATADOG_AGENT ]]; then
+  echo "DISABLE_DATADOG_AGENT environment variable is set, not starting the agent."
+  exit 0
+fi
+
+# Update integration directory configuration
+sed -i -e"s/^.*UNIX_CONFIG_PATH =.*$/UNIX_CONFIG_PATH = '\/app\/.apt\/opt\/datadog-agent\/agent'/" /app/.apt/opt/datadog-agent/agent/config.py
```

Comment:
to be 100 % sure, I will try the buildpack without this change to see if the datadog will stop receiving data.